mypy/checkmember.py

```diff
@@ -378,6 +378,9 @@ def analyze_class_attribute_access(itype: Instance,
         not_ready_callback(name, context)
         return AnyType()
 
+    if isinstance(node.node, TypeVarExpr):
+        return node.node.upper_bound
```

Comment:
If we don't support it, this should also report an error message. (In error recovery mode, returning the upper bound sounds fine.)